generators/server/templates/src/main/resources/config/_application.yml

```diff
@@ -123,7 +123,7 @@ jhipster:
         corePoolSize: 2
         maxPoolSize: 50
         queueCapacity: 10000
-    #cors: #By default CORS are not enabled. Uncomment to enable.
+    #cors: #By default CORS is disabled. Uncomment to enable.
```

Comment:
This will need to be changed in the jhipster mvn library (and would need a release), I can do a PR later tonight if you prefer that way.